games/o.yaml

```
@@ -486,6 +486,26 @@
   status: unplayable
   type: remake
   updated: 2015-07-17
+  
+- name: OpenDominion
+  type: clone
+  originals:
+  - Dominion
+  repo: https://github.com/WaveHack/OpenDominion
+  url: https://beta.opendominion.net/
+  development: active
```

```suggestion
  development: sporadic
```
https://github.com/WaveHack/OpenDominion/commits/develop
progress seems to have slowed lately

---

Hi there. Creator of OpenDominion here.

You're right. Progress has been sporadic as of late, due to numerous (real-life) issues. Thank you for updating it!